depfinder/cli.py

```diff
@@ -135,9 +171,15 @@ def dump_deps(deps):
         deps : dict
             Dictionary of dependencies that were found
         """
+        if keys is None:
+            keys = list(deps.keys())
```

Comment:
Do you need a list? Iteration should work on the next line with a py3 iterator instead from just `deps.keys()`.